.travis.yml

```diff
@@ -24,12 +24,12 @@ matrix:
     - env: EMBER_TRY_SCENARIO=ember-canary
 
 before_install:
-  - npm config set spin false
-  - npm install -g bower
-  - npm install phantomjs-prebuilt
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
I think we want this to crash on CI in case we're doing anything that's not compatible with a new yarn version.